R/utilities.R

```diff
@@ -47,6 +49,76 @@ processArguments <- function(x,
     x
 }
 
+checkMultipleDataSets <- function(x, function.name)
+{
+    variables <- lapply(x, function(x) if (isVariable(x)) x)
+    variables <- Filter(Negate(is.null), variables)
+    if (length(variables) != 0L)
+    {
+        datasets <- unique(vapply(variables, attr, character(1L), which = "dataset"))
+        datasets <- unique(datasets)
```

Comment:
`unique` was already called on the previous line

Reply:
Fixed in https://github.com/Displayr/verbs/pull/12/commits/bb71d95f7a7024a755c0d1179c7dbc45c51af360